lib/src/helpers/authentication-helper.ts

```diff
@@ -149,6 +156,76 @@ export class TokenExchangeAuthenticationHelper<
         }
     }
 
+    public async refreshStsAccessToken(): Promise<void> {
+        const config =
+            (await this._dataLayer.getConfigData()) as AuthClientConfig<CustomAuthClientConfig>;
+
+        const stsSessionData: StsExchangeResponse =
+            await this._dataLayer.getCustomData<StsExchangeResponse>(
+                StsStore.SessionData
+            );
+
+		if (
+			!stsSessionData?.refresh_token
```

Comment:
Shall we get this conditional statement to one line?

Reply:
Fixed the if condition to oneline.